oregon/static/titlecities.txt

```diff
@@ -0,0 +1,62 @@
+label,lat,lon,url
```

Comment:
Is there any reason we couldn't put this data into `core_place`, and generate this file via a django-admin script?  (Perhaps something we add to the compile.sh script even) I'd prefer a single source for all city data, but I'm not sure if there was some reason for not choosing that table.